strapi/strapi
# [RTE]: Block renderers

### What does it do?

Added block renderers for below elements:

- Heading from 1 to 6
- Bold, Italic, Underline, Strikethrough, inline code
- Bullet List
- Number List
- Code 
- Image 
- Link 
- Quote

### Why is it needed?
https://strapi-inc.atlassian.net/browse/CS-191

### How to test it?
Go to CTB, create a new "Block" field and check block input field under CM. 
For now schema is hardcoded on front, so user will see formatted data in block input.
